sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -364,3 +364,38 @@ task validatesContainer() {
     }
   }
 }
+
+def tensorRTTests = tasks.create("tensorRTtests") {
+  dependsOn 'installGcpTest'
+  dependsOn ':sdks:python:sdist'
+ doLast {
+  def testOpts = basicPytestOpts
+  def argMap = [
+    "runner": "DataflowRunner",
+    "machine_type":"n1-standard-4",
+    "sdk_container_image": "us.gcr.io/apache-beam-testing/python-postcommit-it/tensor_rt:latest",
```

Comment:
@AnandInguva @yeandy can we add the Dockerfile for this image somewhere?

Reply:
I can paste the contents of the Dockerfile we used in a comment in https://github.com/apache/beam/issues/22651

Reply:
@tvalentyn Ideas on a good location? I was going to just upload to `apache_beam/ml/inference/` directory, if that's ok.

Reply:
Do we do this for any other tests yet/do we have a pattern we can follow? If not, I'd probably vote we put it in a `test-suites/dataflow/containers` directory and add a small README in that folder that describes that it holds images used to run Dataflow tests. That way we have a reusable pattern for similar tests in the future.

Also btw, I tested the current implementation and it all worked so we should be good to merge this once the Dockerfile is in.

Reply:
no precedents, i like @damccorm 's idea. Could use: test-suites/dataflow/containers/tensortt_runinference

Reply:
actually, they could also be used for other runners, so could be a little up the tree, such as test-suites/containers/tensortt_runinference

Reply:
Added `sdks/python/test-suites/containers/tensorrt_runinference/tensor_rt.dockerfile` and `sdks/python/test-suites/containers/tensorrt_runinference/README.md`

Reply:
Looks like they're missing apache headers

Reply:
I added them, I'll merge once checks pass

Reply:
Thanks!